pyquil/wavefunction.py

```diff
@@ -101,6 +101,10 @@ def __setitem__(self, key, value):
     def __str__(self):
         return self.pretty_print(decimal_digits=10)
 
+    def probabilities(self):
+        """Returns an array of probabilities in lexicographical order"""
+        return np.abs(self.amplitudes ** 2)
```

Comment:
mega nitpicky, but you can save a tiny bit of work by first taking the absolute value and then squaring.

Reply:
btw. I am not sure `np.abs` is any different from `abs`.

Reply:
oops I meant to do `np.abs(amplitudes)**2`. The docstring for `np.abs` suggests it does the correct thing for complex values